README.md

```diff
@@ -75,17 +79,21 @@ Substrate's canonical mnemonic: `bottom drive obey lake curtain smoke basket hol
 - PrivKey:0x96b8a38e12e1a31dee1eab2fffdf9d9990045f5b37e44d8cc27766ef294acf18
 ```
 
+You can find the full list of pre-funded account [here](https://docs.moonbeam.network/getting-started/local-node/setting-up-a-node/#pre-funded-development-accounts)
```

Comment:
```suggestion
You can find the [full list of pre-funded accounts](https://docs.moonbeam.network/getting-started/local-node/setting-up-a-node/#pre-funded-development-accounts) in our docs site.
```

Although it is weird that we have a partial list here. I didn't realize there were more. Should we just have that list in one place?

Reply:
Updated it to list all prefunded accounts